spyder/plugins/preferences/api.py

```diff
@@ -460,22 +475,32 @@ def show_message(is_checked=False):
                                                 msg_info, QMessageBox.Ok)
             checkbox.clicked.connect(show_message)
         checkbox.restart_required = restart
-        return checkbox
+        widget = QWidget(self)
+        widget.checkbox = checkbox
+        if tip is not None:
+            if help_info:
+                layout, help_label = self.add_help_info_label(layout, tip)
+                widget.help_label = help_label
+            else:
+                checkbox.setToolTip(tip)
```

Comment:
This is extra complexity that I think it's not necessary. We want all tooltips in Preferences to be shown as help info labels so they are easily discoverable.

So, please remove the `help_info` kwarg from all methods here and leave this kind of `if` statements as:

```python
if tip is not None:
    layout, help_label = self.add_help_info_label(layout, tip)
    widget.help_label = help_label
```